operator/config/docs/config.json

```
@@ -4,7 +4,9 @@
     ],
     "hideTypePatterns": [
         "ParseError$",
-        "List$"
+        "List$",
+        "ControllerManagerConfiguration$",
+        "ControllerManagerConfigurationSpec$"
```

I wonder if we should just add all the types that do not describe feature-gates here?

```suggestion
        "ControllerHealth$",
        "ControllerManagerConfiguration$",
        "ControllerManagerConfigurationSpec$",
        "ControllerMetrics$",
        "ControllerWebhook$",
        "ProjectConfig$"
```